docs/proposals/multiple-schedulers.md

```diff
@@ -152,6 +152,12 @@ which the client has set a scheduler annotation that does not correspond to a ru
 2. Dynamic launching scheduler(s) and registering to admission controller (as an external call). This also
 requires some work on authorization and authentication to control what schedulers can write the /binding
 subresource of which pods.
+3. Optimize the behaviors of priority functions in multi-scheduler scenario. Running multi-scheduler would make the
+priority functions related to resource allocation less meaningful/efficient. Since all schedulers would tend to pick
+the node with most free resource available. We can pass a parameter such as `experimental-multi-schedulde-mode=true`
+to scheduler, to disable some of the priority functions(such as `LeastRequestedPriority`). Another solution is to
```

Comment:
I think you would not want to disable priority functions, so please remove that part.

Randomizing is reasonable. I would suggest calling the flag something like `randomize-node-selection`. The parameter value would be the `N` you mentioned (so setting this flag would cause the scheduler to pick randomly among the top N nodes).

Another possible parameterization would be with respect to the priority score itself (e.g. setting the flag to 1.5 would mean pick randomly among all nodes that are within 1.5 priority units of the top-priority node) but I think we'd like to move away from the numerical priority approach so let's not do that.